github_is_so_cool/ReadMe.md

```diff
@@ -0,0 +1,3 @@
+[Github](https://www.github.com) is one of the coolest thing a software developer could think of.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.